### ISSUE TITLE
[Enhancement] enable snooze from lockscreen

### ISSUE THREAD

**Issue by gempickfordwaugh** [MilosKozak/AndroidAPS#2445]
Enable both clearer notification of errors and ability to snooze or silence the loud alarm for a missed bolus from the lockscreen of the phone without having to go into the app.  Where users have started delivering a bolus and walked away from their phone before completion the phone thinks bolus is not delivered (quite rightly) but other people such as co-workers etc cannot see the error message or quiet it - the user knows it is not something of concern.  But equally if it was something of concern then other people can't troubleshoot what is causing the noise and whether they need to support the diabetic or not due to lack of info on lockscreen and they won't know the password/pin/pattern to get in..

**Comment by jamespaulley** [comment]
Yeah, cool. Often this happens to me after disrupted bluetooth connection during a bolus and the only way to silence the alarm is to Force Stop AAPS, which is annoying.
